lib/models/ObjectMD.js

```diff
@@ -706,7 +707,7 @@ class ObjectMD {
      */
     setReplicationInfo(replicationInfo) {
         const { status, backends, content, destination, storageClass, role,
-            storageType, dataStoreVersionId } = replicationInfo;
+            storageType, dataStoreVersionId, isNFS } = replicationInfo;
```

Comment:
`fromNFS` could read better, along with `setReplicationFromNFS()`?

Reply:
we "cross repository" agreed on "isNFS" to carry the fact that a bucket might be exported with NFS.